From 1bbf07805df073527a88ada647430adf459d1574 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Sun, 22 Dec 2024 18:55:39 +0000
Subject: [PATCH] Remove parts for wayland vulkan

Upstream-Status: Inappropriate

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 platform/linux/linux.cmake | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/platform/linux/linux.cmake b/platform/linux/linux.cmake
index e1374d067a..8bbbafabf0 100644
--- a/platform/linux/linux.cmake
+++ b/platform/linux/linux.cmake
@@ -15,18 +15,6 @@ find_package(Threads REQUIRED)
 pkg_search_module(WEBP libwebp REQUIRED)
 pkg_search_module(LIBUV libuv REQUIRED)
 
-if(MLN_WITH_WAYLAND)
-    # See https://github.com/maplibre/maplibre-native/pull/2022
-
-    # MLN_WITH_EGL needs to be set for Wayland, otherwise this CMakeLists will
-    # call find_package(OpenGL REQUIRED GLX), which is for X11.
-    set(MLN_WITH_EGL TRUE)
-
-    # OPENGL_USE_GLES2 or OPENGL_USE_GLES3 need to be set, otherwise
-    # FindOpenGL.cmake will include the GLVND library, which is for X11.
-    set(OPENGL_USE_GLES3 TRUE)
-endif()
-
 target_sources(
     mbgl-core
     PRIVATE
@@ -174,7 +162,6 @@ target_link_libraries(
 
 add_subdirectory(${PROJECT_SOURCE_DIR}/bin)
 add_subdirectory(${PROJECT_SOURCE_DIR}/expression-test)
-add_subdirectory(${PROJECT_SOURCE_DIR}/platform/glfw)
 if(MLN_WITH_NODE)
     add_subdirectory(${PROJECT_SOURCE_DIR}/platform/node)
 endif()
-- 
2.47.1

