From 151ba7108360c0f1c0ada836eea7a7e43022f3e7 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Sun, 22 Dec 2024 18:49:54 +0000
Subject: [PATCH] Remove deps

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 platform/linux/linux.cmake | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/platform/linux/linux.cmake b/platform/linux/linux.cmake
index e1374d067a..08a49221a4 100644
--- a/platform/linux/linux.cmake
+++ b/platform/linux/linux.cmake
@@ -15,18 +15,6 @@ find_package(Threads REQUIRED)
 pkg_search_module(WEBP libwebp REQUIRED)
 pkg_search_module(LIBUV libuv REQUIRED)
 
-if(MLN_WITH_WAYLAND)
-    # See https://github.com/maplibre/maplibre-native/pull/2022
-
-    # MLN_WITH_EGL needs to be set for Wayland, otherwise this CMakeLists will
-    # call find_package(OpenGL REQUIRED GLX), which is for X11.
-    set(MLN_WITH_EGL TRUE)
-
-    # OPENGL_USE_GLES2 or OPENGL_USE_GLES3 need to be set, otherwise
-    # FindOpenGL.cmake will include the GLVND library, which is for X11.
-    set(OPENGL_USE_GLES3 TRUE)
-endif()
-
 target_sources(
     mbgl-core
     PRIVATE
-- 
2.47.1

