diff --git a/filament/CMakeLists.txt b/filament/CMakeLists.txt
index fae6565..3bbb22b 100644
--- a/filament/CMakeLists.txt
+++ b/filament/CMakeLists.txt
@@ -651,7 +651,7 @@ target_link_libraries(${TARGET} PRIVATE
 # ==================================================================================================
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/filament DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/filament DESTINATION include/filament)
 install(FILES "README.md" DESTINATION .)
 install(FILES "../LICENSE" DESTINATION .)
 
diff --git a/filament/backend/CMakeLists.txt b/filament/backend/CMakeLists.txt
index 24cc789..4bfd9c5 100644
--- a/filament/backend/CMakeLists.txt
+++ b/filament/backend/CMakeLists.txt
@@ -493,7 +493,7 @@ target_link_libraries(${TARGET} PRIVATE
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
 install(TARGETS vkshaders ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/backend DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/backend DESTINATION include/filament)
 
 if (FILAMENT_SUPPORTS_WEBGPU)
     install(TARGETS webgpu_dawn ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
diff --git a/libs/camutils/CMakeLists.txt b/libs/camutils/CMakeLists.txt
index 541261c..013cbce 100644
--- a/libs/camutils/CMakeLists.txt
+++ b/libs/camutils/CMakeLists.txt
@@ -45,7 +45,7 @@ endif()
 # Installation
 # ==================================================================================================
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/camutils DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/camutils DESTINATION include/filament)
 
 # ==================================================================================================
 # Tests
diff --git a/libs/filabridge/CMakeLists.txt b/libs/filabridge/CMakeLists.txt
index 6840ce5..9627713 100644
--- a/libs/filabridge/CMakeLists.txt
+++ b/libs/filabridge/CMakeLists.txt
@@ -32,5 +32,5 @@ target_link_libraries(${TARGET} backend_headers)
 # ==================================================================================================
 # Installation
 # ==================================================================================================
-install(DIRECTORY ${PUBLIC_HDR_DIR}/filament DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/filament DESTINATION include/filament)
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
diff --git a/libs/filamat/CMakeLists.txt b/libs/filamat/CMakeLists.txt
index 3d790b0..dd397df 100644
--- a/libs/filamat/CMakeLists.txt
+++ b/libs/filamat/CMakeLists.txt
@@ -144,7 +144,7 @@ combine_static_libs(filamat "${FILAMAT_COMBINED_OUTPUT}" "${FILAMAT_DEPS}")
 
 set(FILAMAT_LIB_NAME ${CMAKE_STATIC_LIBRARY_PREFIX}filamat${CMAKE_STATIC_LIBRARY_SUFFIX})
 install(FILES "${FILAMAT_COMBINED_OUTPUT}" DESTINATION lib/${DIST_DIR} RENAME ${FILAMAT_LIB_NAME})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/filamat DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/filamat DESTINATION include/filament)
 
 # Need to install libtint for filamat on Android.
 # See libs/filamat/CMakeLists.txt and android/filamat-android/CMakeLists.txt.
diff --git a/libs/filameshio/CMakeLists.txt b/libs/filameshio/CMakeLists.txt
index eab3192..7143002 100644
--- a/libs/filameshio/CMakeLists.txt
+++ b/libs/filameshio/CMakeLists.txt
@@ -32,7 +32,7 @@ target_link_libraries(${TARGET}
 # ==================================================================================================
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
-install(FILES ${DIST_HDRS} DESTINATION include/${TARGET})
+install(FILES ${DIST_HDRS} DESTINATION include/filament/${TARGET})
 
 # ==================================================================================================
 # Tests
diff --git a/libs/geometry/CMakeLists.txt b/libs/geometry/CMakeLists.txt
index a7e8f39..5a5bb88 100644
--- a/libs/geometry/CMakeLists.txt
+++ b/libs/geometry/CMakeLists.txt
@@ -55,7 +55,7 @@ endif()
 # No need to install since we're combining this lib and the dependent libs into a combined lib
 # install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
 
-install(DIRECTORY ${PUBLIC_HDR_DIR}/geometry DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/geometry DESTINATION include/filament)
 
 set(COMBINED_DEPS
     ${TARGET}
diff --git a/libs/gltfio/CMakeLists.txt b/libs/gltfio/CMakeLists.txt
index 373df1f..de289cf 100644
--- a/libs/gltfio/CMakeLists.txt
+++ b/libs/gltfio/CMakeLists.txt
@@ -265,5 +265,5 @@ endif()
 # ==================================================================================================
 
 install(TARGETS gltfio_core uberarchive ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/gltfio DESTINATION include)
-install(FILES ${RESOURCE_DIR}/materials/uberarchive.h DESTINATION include/gltfio/materials)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/gltfio DESTINATION include/filament)
+install(FILES ${RESOURCE_DIR}/materials/uberarchive.h DESTINATION include/filament/gltfio/materials)
diff --git a/libs/iblprefilter/CMakeLists.txt b/libs/iblprefilter/CMakeLists.txt
index c001de6..62066b3 100644
--- a/libs/iblprefilter/CMakeLists.txt
+++ b/libs/iblprefilter/CMakeLists.txt
@@ -135,4 +135,4 @@ target_link_libraries(${TARGET} PRIVATE
 # ==================================================================================================
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/filament-iblprefilter DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/filament-iblprefilter DESTINATION include/filament)
 
 # ==================================================================================================
 # Tests
diff --git a/libs/ktxreader/CMakeLists.txt b/libs/ktxreader/CMakeLists.txt
index a9d5714..039ee6a 100644
--- a/libs/ktxreader/CMakeLists.txt
+++ b/libs/ktxreader/CMakeLists.txt
@@ -41,7 +41,7 @@ endif()
 # Installation
 # ==================================================================================================
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/${TARGET} DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/${TARGET} DESTINATION include/filament)
 
 # ==================================================================================================
 # Tests
diff --git a/libs/math/CMakeLists.txt b/libs/math/CMakeLists.txt
index 838aa2d..4256d66 100644
--- a/libs/math/CMakeLists.txt
+++ b/libs/math/CMakeLists.txt
@@ -47,7 +47,7 @@ set_target_properties(${TARGET} PROPERTIES FOLDER Libs)
 # ==================================================================================================
 # Installation
 # ==================================================================================================
-install(DIRECTORY ${PUBLIC_HDR_DIR}/math DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/math DESTINATION include/filament)
 
 # ==================================================================================================
 # Tests
diff --git a/libs/mathio/CMakeLists.txt b/libs/mathio/CMakeLists.txt
index 4ad5126..96b5287 100644
--- a/libs/mathio/CMakeLists.txt
+++ b/libs/mathio/CMakeLists.txt
@@ -27,5 +27,5 @@ target_link_libraries(${TARGET} PRIVATE math)
 # ==================================================================================================
 # Installation
 # ==================================================================================================
-install(DIRECTORY ${PUBLIC_HDR_DIR}/mathio DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/mathio DESTINATION include/filament)
 
diff --git a/libs/uberz/CMakeLists.txt b/libs/uberz/CMakeLists.txt
index 47744e2..85569b5 100644
--- a/libs/uberz/CMakeLists.txt
+++ b/libs/uberz/CMakeLists.txt
@@ -44,4 +44,4 @@ endif()
 # Installation
 # ==================================================================================================
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/uberz DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/uberz DESTINATION include/filament)
diff --git a/libs/utils/CMakeLists.txt b/libs/utils/CMakeLists.txt
index 3af98d0..ebb02e4 100644
--- a/libs/utils/CMakeLists.txt
+++ b/libs/utils/CMakeLists.txt
@@ -139,11 +139,11 @@ endif()
 # ==================================================================================================
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
-install(FILES ${DIST_HDRS} DESTINATION include/${TARGET})
+install(FILES ${DIST_HDRS} DESTINATION include/filament/${TARGET})
 if (ANDROID)
-    install(FILES ${DIST_ANDROID_HDRS} DESTINATION include/${TARGET_LINUX})
+    install(FILES ${DIST_ANDROID_HDRS} DESTINATION include/filament/${TARGET_LINUX})
 else()
-    install(FILES ${DIST_GENERIC_HDRS} DESTINATION include/${TARGET_GENERIC})
+    install(FILES ${DIST_GENERIC_HDRS} DESTINATION include/filament/${TARGET_GENERIC})
 endif()
 
 # ==================================================================================================
diff --git a/libs/viewer/CMakeLists.txt b/libs/viewer/CMakeLists.txt
index 3b67fde..5e40ef1 100644
--- a/libs/viewer/CMakeLists.txt
+++ b/libs/viewer/CMakeLists.txt
@@ -54,7 +54,7 @@ endif()
 # Installation
 # ==================================================================================================
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/viewer DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/viewer DESTINATION include/filament)
 
 # ==================================================================================================
 # Tests
diff --git a/third_party/libassimp/contrib/zip/CMakeLists.txt b/third_party/libassimp/contrib/zip/CMakeLists.txt
index b46dbb1..368017a 100644
--- a/third_party/libassimp/contrib/zip/CMakeLists.txt
+++ b/third_party/libassimp/contrib/zip/CMakeLists.txt
@@ -33,7 +33,7 @@ install(TARGETS ${PROJECT_NAME}
         ARCHIVE DESTINATION lib
         LIBRARY DESTINATION lib
         COMPONENT library)
-install(FILES ${PROJECT_SOURCE_DIR}/src/zip.h DESTINATION include)
+install(FILES ${PROJECT_SOURCE_DIR}/src/zip.h DESTINATION include/filament)
 
 # uninstall target (https://gitlab.kitware.com/cmake/community/wikis/FAQ#can-i-do-make-uninstall-with-cmake)
 if(NOT TARGET uninstall)
diff --git a/third_party/libsdl2/CMakeLists.txt b/third_party/libsdl2/CMakeLists.txt
index 8f1e828..b1d7cbd 100644
--- a/third_party/libsdl2/CMakeLists.txt
+++ b/third_party/libsdl2/CMakeLists.txt
@@ -1806,7 +1806,7 @@ foreach(_FNAME ${BIN_INCLUDE_FILES})
   list(REMOVE_ITEM INCLUDE_FILES ${SDL2_SOURCE_DIR}/include/${_INCNAME})
 endforeach()
 list(APPEND INCLUDE_FILES ${BIN_INCLUDE_FILES})
-install(FILES ${INCLUDE_FILES} DESTINATION include/SDL2)
+install(FILES ${INCLUDE_FILES} DESTINATION include/filament/SDL2)
 
 if(NOT (WINDOWS OR CYGWIN))
   if(SDL_SHARED)
diff --git a/third_party/mikktspace/CMakeLists.txt b/third_party/mikktspace/CMakeLists.txt
index 36959ae..6c8fead 100644
--- a/third_party/mikktspace/CMakeLists.txt
+++ b/third_party/mikktspace/CMakeLists.txt
@@ -26,4 +26,4 @@ endif()
 # Installation
 # ==================================================================================================
 install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
-install(DIRECTORY ${PUBLIC_HDR_DIR}/mikktspace DESTINATION include)
+install(DIRECTORY ${PUBLIC_HDR_DIR}/mikktspace DESTINATION include/filament)
diff --git a/third_party/robin-map/tnt/CMakeLists.txt b/third_party/robin-map/tnt/CMakeLists.txt
index 300da52..ef5ef82 100644
--- a/third_party/robin-map/tnt/CMakeLists.txt
+++ b/third_party/robin-map/tnt/CMakeLists.txt
@@ -27,4 +27,4 @@ set_target_properties(${TARGET} PROPERTIES FOLDER ThirdParty)
 # ==================================================================================================
 # Installation
 # ==================================================================================================
-install(FILES ${PUBLIC_HDRS} DESTINATION include/${TARGET})
+install(FILES ${PUBLIC_HDRS} DESTINATION include/filament/${TARGET})
