From 3f560a0990ec2c681ad38cb07203fa319acc75dd Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Sat, 21 Jan 2023 13:27:05 -0800
Subject: [PATCH] vulkan_hpp crosscompile

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 35813c5..6876f24 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -1,7 +1,7 @@
 prog_python = find_program('python3')
 
 vulkan_hpp = join_paths([
-    vulkan_dep.get_pkgconfig_variable('includedir'),
+    vulkan_dep.get_pkgconfig_variable('includedir', define_variable: ['prefix', get_option('prefix')]),
     'vulkan',
     'vulkan.hpp'
     ])
-- 
2.38.1
