From b32c67f6cc6ca889d5f2d6dc11429ca361bc2856 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Fri, 20 Jan 2023 08:57:18 -0800
Subject: [PATCH] Remove strip step

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 Quake/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Quake/Makefile b/Quake/Makefile
index c35c930..dd20dec 100644
--- a/Quake/Makefile
+++ b/Quake/Makefile
@@ -93,7 +93,7 @@ all: $(DEFAULT_TARGET)
 
 vkquake: $(SHADER_OBJS) $(OBJS)
 	$(LINKER) $(SHADER_OBJS) $(OBJS) $(LDFLAGS) $(LIBS) $(SDL_LIBS) -o $@
-	$(call DO_STRIP,$@)
+
 
 release: vkquake
 debug:
-- 
2.38.1

